browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -230,6 +233,30 @@ class MarkdownNoteDetail extends React.Component {
     }
   }
 
+  handleFullScreenButton (e) {
+    const currentScreenState = !Object.assign({}, this.state).fullScreen
```

Comment:
Now that you don't have to use `Object.assign` (I commented that before I offer you to change the structure of the state).
You can simply write `this.setState({ fullScreen: !this.state.fullScreen })`.